sdk/core/src/request_options/rename_source.rs

```diff
@@ -0,0 +1,42 @@
+use crate::headers::*;
+use crate::AddAsHeader;
+use http::request::Builder;
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct RenameSource<'a>(&'a str);
```

Comment:
Is this used anywhere but in storage?

Also, it would be nice to have a short doc comment description of what this is

```
An optional file or directory to be renamed.
```

Reply:
Good point, let me check, but probably only used in data_lake. Will add a comment as well.